src/crimson/osd/osd_operations/peering_event.cc

```diff
@@ -57,7 +57,7 @@ seastar::future<> PeeringEvent::start()
   auto maybe_delay = seastar::now();
   if (delay) {
     maybe_delay = seastar::sleep(
-      std::chrono::milliseconds(std::lround(delay * 1000)));
+      std::chrono::milliseconds(std::lround(delay)));
```

Comment:
This changes the meaning of PG::schedule_event_after, but it doesn't currently have any users, so that's ok.

Reply:
remove this change, and set delay to be 0.001